### PR TITLE
Improve page titles

### PR DESCRIPTION
**Pull request by majjikishore007**
**Closes:** https://github.com/tramlinehq/site/issues/580

**Because:**  
We want our page titles to be more meaningful than "ORG_NAME | Tramline."

**This Addresses:**  
- Adds an I18n-based setup in `en.yml` so we have more user-friendly, localized page titles.
- Centralizes all title logic for making updates and maintenance simpler

Quick demo :  
https://www.loom.com/share/b4c3ad5e3e2d4961b3e045d7bb23756d?sid=a88ee43a-c3c4-4296-9019-10fa8b8bbbb1

### CODE REVIEW

**Comment by kitallis** [comment]
@majjikishore007 Thank you for this PR, give us a few days this week to review it.

**Review by nid90 — CHANGES_REQUESTED** [review]
The PR overall looks good, @majjikishore007! Thank you. 🙇‍♀️ 

I have noted a slight improvement at one place, and some missing locales.

I will merge this once those are resolved!

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -40,6 +40,27 @@ def current_user
     @current_user ||= current_email_authentication&.user || @current_sso_user
   end
 
+  private
+
+  def set_page_name
```
[the remark this answers is not in the document]
Comment (majjikishore007) [reply app/controllers/application_controller.rb]: Hi @nid90, the requested changes were done thanks

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,4 +1,70 @@
...
+          new: "Integration New"
+          edit: "Integration Edit"
+      # ReleaseHealthRules Controller
+      release_health_rules:
+        name: "Release Health Rules"
+        actions:
+          index: "Rules"
+          show: "Rule Details"
+          new: "Rule New"
+          edit: "Rule Edit"
```
[the remark this answers is not in the document]
Comment (majjikishore007) [reply config/locales/en.yml:67]: added the controllers to the config

**Comment by nid90** [comment]
@majjikishore007 Looks good! There is a minor lint issue that can be fixed by running `just lint`. I will merge this once all the checks pass! ⭐

**Comment by majjikishore007** [comment]
@nid90, I have fixed the lint issues thanks

**Review by nid90 — APPROVED** [review]

**Comment by kitallis** [comment]
@majjikishore007 Your PR is ready to be merged, but before we do so, we'd like you to sign our [Contributor Agreement](https://drive.google.com/file/d/1-sE783R8eAn9TGwkDSrjqhZvxQfvVec5/view?usp=sharing). It's a standard [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) based on the Clojure/Eclipse CA.

Please fill and send to contributors AT tramline DOT app

**Comment by majjikishore007** [comment]
@kitallis I have signed and sent the document to the mentioned email address. do let me know if anything needs to be added more, thanks

**Pull request merged**